### PR TITLE
Display artwork when an audio file is reproduced

### PR DESCRIPTION
**Pull request by andrevz**
The constraint for the image is based on the bounds of the parent view (content overlay view)

### CODE REVIEW

File: Owncloud iOs Client/Files/Preview/FilePreview/FilePreviewViewController.m [hunk 1]
```diff
@@ -1141,12 +1141,25 @@ - (void)playMediaFile{
         }
         
         AVPlayer *player;
+        UIImage *image;
```
Comment (hosy) [inline Owncloud iOs Client/Files/Preview/FilePreview/FilePreviewViewController.m]: would be better to initialize variables with `nil` value

File: Owncloud iOs Client/Files/Preview/FilePreview/FilePreviewViewController.m [hunk 2]
```diff
@@ -1172,6 +1185,59 @@ - (void)playMediaFile{
         
         [self.avMoviePlayer.contentOverlayView addObserver:self forKeyPath:[MediaAVPlayerViewController observerKeyFullScreen] options:NSKeyValueObservingOptionNew | NSKeyValueObservingOptionOld context:NULL];
         
+
+        if (image != nil) {
+            UIImageView *imageView = [[UIImageView alloc] initWithFrame:CGRectMake(0,0,300,300)];
```
Comment (hosy) [inline Owncloud iOs Client/Files/Preview/FilePreview/FilePreviewViewController.m]: Is the width and height a static size?

**Comment by hosy** [comment]
@andrevz thank you very much for your contribution!

File: Owncloud iOs Client/Files/Preview/FilePreview/FilePreviewViewController.m (hunk 2 above)
Comment (andrevz) [reply Owncloud iOs Client/Files/Preview/FilePreview/FilePreviewViewController.m]: Yep, but it is auto resized and centered based on NSLayoutConstraints, any idea how to initialize without the hardcoded sizes?

**Comment by mmattel** [comment]
@nasli a second review is necessery, do you mind? :smile:

**Comment by hosy** [comment]
@andrevz please could you sign the https://cla-assistant.io/owncloud/ios?pullRequest=1105 ?

**Pull request merged**